frank-doc-doclet/pom.xml

```diff
@@ -8,7 +8,7 @@
 	</parent>
 
 	<artifactId>frank-doc-doclet</artifactId>
-	<version>1.2-SNAPSHOT</version>
+	<version>${project.parent.version}</version>
```

Comment:
```suggestion
```
Dit moet weg, en in de parent moet (middels het dependencyManagement stuk) de versie voor elke module bepaald worden.